src/pint/utils.py

```diff
@@ -1667,3 +1673,90 @@ def remove_dummy_distance(c):
             "Do not know coordinate frame for %r: returning coordinates unchanged" % c
         )
         return c
+
+
+def info_string(prefix_string="# ", comment=None):
+    """
+    Returns an informative string about the current state of PINT.
```

Comment:
This should be on the first line of the docstring (same line as the triple-quote) according to docstring standards.

Reply:
fixed.